Makefile

```diff
@@ -22,7 +22,7 @@ dist:
 
 test:
 	@echo "--> Running go test"
-	@go test $(PACKAGES)
+	@go test -v $(PACKAGES)
```

Comment:
I prefer to not have this because there's too much output when you just want to see a bunch of "ok"